index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header class="navbar">
+       <a class="logo">My Logo</a>
+       <nav>
+         <img src="Union.png" alt="menubar">
+       </nav>
+    </header>
+  
+     <section class="main">
+       <h1 class="header">I’m Ryisa Glad to see you! </h1>
+          <p class="section">
+           I’m a software developer! I can help you build a product,feature or 
+           website Look through some of my work and experience! If you like what 
+           you see and have a project you need coded, don’t hestiate to contact me.
+          </p>
+           <p class="footer">LET CONNECT</p>
+            <ul class="img">
+              <li><a href="#"><img src="twitter.svg" alt="twitter handle"></a></li>
+              <li><a href="#"><img src="github.jpg" alt="github handle"></a></li>
+              <li><a href="#"><img src="medium.jpg" alt="medium handle"></a></li>
+              <li><a href="#"><img src="angellist.jpg" alt="angellisthandle"></a></li>
+            </ul>
+     </section>
```

Comment:
- Your project is missing work section and about section. Kindly work on [the project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) and after that request another code review.